spec/rails_helper.rb

```diff
@@ -8,49 +8,42 @@
 require 'shoulda-matchers'
 require File.join(File.dirname(__FILE__), 'support/valid_attribute')
 require File.join(File.dirname(__FILE__), 'support/factory_girl')
+Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.